addremove.js

```diff
@@ -0,0 +1,30 @@
+import { items, AddItem } from './__mock__/index.js';
+import { setStored, displayList, checkList } from './src/index.js';
+
+const addCrud = () => {
+  const addEnter = document.querySelector('.add-enter');
+  addEnter.addEventListener('keyup', ({ key }) => {
+    if (key === 'Enter' && addEnter.value !== '') {
+      const newEntry = new AddItem(addEnter.value, items.length);
+      items.push(newEntry);
+      addEnter.value = '';
+      checkList.innerHTML = '';
+      setStored();
+      displayList();
+    }
+  });
+};
+
+const removeCrud = (trashIcon) => {
+  trashIcon.addEventListener('click', () => {
+    const filtered = items.filter((item) => item.id !== trashIcon.id);
+    filtered.forEach((item, i) => {
+      item.index = i;
+      item.id = `item-${i}`;
+    });
+    localStorage.setItem('newItems', JSON.stringify(filtered));
+    displayList();
+  });
+};
+
+export { addCrud, removeCrud };
```

Comment:
- [OPTIONAL] Please it will be nice to add tests for more functions to increase your test coverage. See image below.

![add-test](https://user-images.githubusercontent.com/65852621/146381543-c0cd7b77-54e9-4e99-ab95-9760550d1b3d.png)